src/uhs/twophase/sentinel_2pc/controller.cpp

```diff
@@ -25,6 +25,20 @@ namespace cbdc::sentinel_2pc {
                                                    .size())]) {}
 
     auto controller::init() -> bool {
+        if(m_sentinel_id > (m_opts.m_sentinel_endpoints.size() - 1)) {
+            m_logger->error(
+                "The sentinel ID is too large for the number of sentinels.");
+            return false;
+        }
+
+        if(m_opts.m_attestation_threshold
```

Comment:
I think this check more appropriately belongs in the configuration file parsing code where there are other sanity checks like this.

Reply:
Good call.  In the latest commit, I put it with the other sanity checks on the `options` struct in the function `cbdc::config::check_options`.

Reply:
For my own clarity, which check got moved?

Reply:
Pardon the noise. The way github is displaying this is quite strange.